int32 keypoints_count
KeyPoint[] img_keypoints
